src/mumble/GlobalShortcut_win.cpp

```diff
@@ -129,7 +132,8 @@ GlobalShortcutWin::GlobalShortcutWin()
 #endif
 {
 	// Hidden setting to disable hooking
-	bHook = g.qs->value(QLatin1String("winhooks"), true).toBool();
+	// Also disable hooking if a screen reader is running
+	bHook = g.qs->value(QLatin1String("winhooks"), true).toBool() || areScreenReadersActive();
```

Comment:
Shouldn't this be `!areScreenReadersActive()` (note the not-operator)? Because as far as I understood, you want to *disable* winhooks if a ScreenReader is active...

Reply:
Also if you want to force-disable winhooks if a screenreader is active, you'll have to use
```
bHook = g.qs->value(QLatin1String("winhooks"), true).toBool() && !areScreenReadersActive();
```